quantecon/tests/test_lqcontrol.py

```diff
@@ -81,7 +81,7 @@ def test_stationary_mat(self):
         lq_mat = self.lq_mat
 
         f_answer = np.array([[-.95, -.95], [0., 0.]])
-        p_answer = np.array([[1., 0], [0., 0.]])
+        # p_answer = np.array([[1., 0], [0., 0.]])  # unused
```

Comment:
@natashawatkins would you have time to check this?

Reply:
doesn't appear to be used